matchers/utilities.cpp

```diff
@@ -428,6 +428,24 @@ boost::optional<json> DetailFunctionDecl(const hyde::processing_options& options
             break;
     }
 
+	auto visibility = f->getVisibility();
+	switch (visibility) {
+		case Visibility::HiddenVisibility:
+			info["visibility"] = "hidden";
+			break;
+		case Visibility::DefaultVisibility:
+			info["visibility"] = "default";
+			break;
+		case Visibility::ProtectedVisibility:
+			info["visibility"] = "protected";
+			break;
+		default:
```

Comment:
I'd rather there not be a `default` option. That way in case these enumerations change we get a compiler warning about them, instead of values going blank unexpectedly.

Reply:
Done!